hive/src/main/java/com/netease/arctic/hive/op/UpdateHiveFiles.java

```diff
@@ -99,6 +97,16 @@ public void commit() {
       this.partitionToCreate = getCreatePartition(this.partitionToDelete);
     }
 
+    // if no DataFiles to add or delete in Hive location, only commit to iceberg
```

Comment:
I recommend add this if statement  before line 116 to avoid duplicate code at line 101-105 and line 110-114